src/js/components/__tests__/__snapshots__/README-test.js.snap

```diff
@@ -8346,6 +8346,42 @@ div
 \`\`\`
 ## Theme
   
+**global.borderSize**
+
+The size of the border Expects \`string\`.
+
+Defaults to
+
+\`\`\`
+{
+      xsmall: '1px',
+      small: '2px',
+      medium: '4px',
+      large: '12px',
+      xlarge: '24px,
+    }
+\`\`\`
+
+**global.colors.border**
+
+color of tabs border Expects \`string | { dark: string, light: string }\`.
```

Comment:
here you can see for example how noticeable is the missing dot.

Reply:
@ShimiSun  yes much better will do!